Options.py

```diff
@@ -1133,9 +1152,12 @@ class Accessibility(Choice):
 
 
 class ProgressionBalancing(NamedRange):
-    """
-    A system that can move progression earlier, to try and prevent the player from getting stuck and bored early.
-    A lower setting means more getting stuck. A higher setting means less getting stuck.
+    """Move progression items earlier.
```

Comment:
This is not at all accurate to how progression balancing works, so I disagree with this doc comment change.